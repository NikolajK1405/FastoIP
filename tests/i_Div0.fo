fun int main() =
  let x = read(int) in
  4/x