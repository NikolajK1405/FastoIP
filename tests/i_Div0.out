Interpreter error: Divide by zero error second, left operand of / 
                                     must be larger than 0 at line 3, column 5
